test/functional/test_framework/messages.py

```diff
@@ -54,9 +54,6 @@
 def sha256(s):
     return hashlib.new('sha256', s).digest()
 
-def ripemd160(s):
-    return hashlib.new('ripemd160', s).digest()
```

Comment:
When has this last been used? Not sure if we want to remove library functions that are still used in other branches. Same for the key.py changes.

Reply:
Last use was removed in b9f34e84befa7db6ff8c9b92a09d0dfa40388fb7

Reply:
ack